app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb

```diff
@@ -15,5 +15,12 @@ def initialize_inventory_collections
       %i(credentials),
       :builder_params => {:resource => manager}
     )
+
+    collections[:vms] = ::ManagerRefresh::InventoryCollection.new(
+      :model_class => Vm,
+      :arel        => Vm,
+      :strategy    => :local_db_find_references,
+      :manager_ref => [:uid_ems]
```

Comment:
doing non lazy will degrade to SQL query per find, unless you set the references upfront :-(